school/src/main/java/com/hogwarts/school/controller/StudentController.java

```diff
@@ -51,4 +52,8 @@ public ResponseEntity<List<Student>> getStudentByAge(@PathVariable int studentAg
         List<Student> std = studentService.filteringStudentsByAge(studentAge);
         return ResponseEntity.ok(std);
     }
+    @GetMapping("/getAll")
```

Comment:
```suggestion
    @GetMapping
```
не нужно ничего писать в пути - это ошибка REST